compiler/rustc_ast/src/ast.rs

```diff
@@ -2635,6 +2635,7 @@ impl Default for FnHeader {
     }
 }
 
+/// Represents a concrete type and contains information specific to the type of the item.
```

Comment:
An item isn't a type: it's some kind of definition or statement. I'm not quite sure what the best description of it is, but whatever it is, it should go on `struct Item`, not `ItemKind`, which follows the same convention as all the other `*Kind` types.